docs/mkdocs.yml

```
@@ -119,25 +119,27 @@ markdown_extensions:
-    - exospherehost/send-alert: satellites/exospherehost/send-alert.md
-    - exospherehost/send-email: satellites/exospherehost/send-email.md
-  - Cluster: cluster.md
-  - Architecture: architecture.md
-  - API Server:
-    - Overview: api-server/index.md
-    - Swagger: api-server/swagger.md
\ No newline at end of file
+  - Introduction: exosphere/index.md
+  - Getting Started: exosphere/getting-started.md
+  - State Manager Setup: exosphere/state-manager-setup.md
+  - Register Node: exosphere/register-node.md 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)